scripts/docker/rhel/Dockerfile

```diff
@@ -6,6 +6,14 @@
 # Dockerfile that creates a container suitable to build dotnet-cli
 FROM microsoft/dotnet-buildtools-prereqs:rhel-7-rpmpkg-e1b4a89-20175311035359
 
+# Install repository configuration
+RUN curl https://packages.microsoft.com/config/rhel/7/prod.repo > ./microsoft-prod.repo
```

Comment:
3. and i also get error of `key ID be1229cf: NOKEY ` i need to import Microsoft release public key